syntax = "proto3";

option java_multiple_files = true;
option java_package = "io.grpc.examples.kiosk";
option java_outer_classname = "KioskProto";
option objc_class_prefix = "K";

package kiosk;

// The greeting service definition.
service Kiosk {
  rpc GetData (GetDataRequest) returns (GetDataResponse) {}
  rpc ListData (ListDataRequest) returns (ListDataResponse) {}
}

// The request message containing the user's name.
message GetDataRequest {
  string name = 1;
}

message GetDataResponse {
  string name = 1;
}

// The response message containing the greetings
message ListDataRequest {
  string message = 1;
}

message ListDataResponse {
  string message = 1;
}